### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 9

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#185] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252965-9e609e0f-931a-4148-a327-b25d2517f8ac.png)

**Comment by faciolox** [comment]
Ho usato questo ragionamento, correggetemi se sbaglio.
Ho interpretato la domanda come l'unione dei 3 insiemi delle materie.
Per cui la Risposta sarà:
P + A + L - PA - AL - PL + PAL
ovvero
22+15+8-25-19-14+13
che dovrebbe dare come risultato zero.

**Comment by AxnNxs** [comment]
> Ho usato questo ragionamento, correggetemi se sbaglio. Ho interpretato la domanda come l'unione dei 3 insiemi delle materie. Per cui la Risposta sarà: P + A + L - PA - AL - PL + PAL ovvero 22+15+8-25-19-14+13 che dovrebbe dare come risultato zero.

Anzitutto credo che il testo sia errato: il campione di studenti dovrebbe ammontare a 45, non 35, a meno che non vi siano studenti di cui non si conosce il responso.

La risposta credo sia quindi 45, in quanto non si chiede di trovare P + A + L (con P, A, L disgiunti), ovvero l'unione degli studenti che apprezzano o P, o A, o L, ma a quanti piaccia almeno una materia.

Ragionando a livello insiemistico, questo vuol dire che Il campione S (che ammonta a 45 da ipotesi) è dato dal seguente calcolo:
S = P + A + L - PA - AL - PL + PAL;
Il risultato di questo calcolo, che è 0, dovrebbe dirci soltanto che Gli insiemi "P \ (A + L)", "A \ (P + L)", "L \ (A + P)" siano di fatto vuoti perché vi sono elementi soltanto nelle intersezioni.

Pertanto, oltre al PIE, va effettuato il ragionamento "Il campione totale degli studenti che hanno risposto al sondaggio" - "Il campione che non apprezza nessuna materia, ovvero 0":

45 - 0 = 45. Tutti gli studenti apprezzano almeno una materia.




Risposta 2:
Ammettendo che sia giusto il dato del testo S = 35, potremmo assumere che i 10 studenti mancanti siano proprio quelli che non hanno espresso alcuna preferenza. A quel punto, la risposta coinciderebbe con 35, ovvero, aggiungendo 10 allo 0 trovato prima, avendo scoperto che a tutti gli interpellati piace sicuramente una o più materie, otterremmo che 45 - 10 = 35).
